test/e2e/e2e_suite_test.go

```diff
@@ -187,6 +199,36 @@ func (acp *AzureClusterProxy) collectPodLogs(ctx context.Context, namespace stri
 				}
 			}(pod, container)
 		}
+
+		go func(pod corev1.Pod) {
+			defer GinkgoRecover()
+
+			Byf("Collecting events for Pod %s/%s", kubesystem, pod.Name)
+			eventFile := path.Join(aboveMachinesPath, kubesystem, pod.Name, "pod-events")
```

Comment:
Should we use a file extension here, like `pod-events.txt` or `pod-events.log`? It can help with inferring mime types and telling a user what contents to expect.

Reply:
Go with `.txt` since we already have `.log` and they are different. Thank you!